cloud/linode/vpc.go

```diff
@@ -3,11 +3,18 @@ package linode
 import (
 	"context"
 	"fmt"
+	"sync"
 
 	"github.com/linode/linode-cloud-controller-manager/cloud/linode/client"
 	"github.com/linode/linodego"
 )
 
+var (
+	Mu sync.RWMutex
+	// vpcIDs map stores vpc id's for given vpc labels
+	vpcIDs = make(map[string]int, 0)
```

Comment:
Do we want to have timeout for the stored vpc ids? I wonder if we'll need to refresh it in case a VPC gets deleted and re-created or just deleted.

Reply:
I think doing that and having a cache here makes sense to cover that edge case

Reply:
updated the code to remove vpcid if it doesn't exists. In case of vpc deletion or re-creation, id will change and we can remove stale ones.